app/controllers/webhooks_controller.rb

```diff
@@ -33,7 +33,7 @@ def handle_unknown_recipient!
 
   def handle_blacklisted!
     if BLACKLISTED_SENDERS.include? parsed_email.from.local.downcase
-      logger.error "Blacklisted sender detected. Skipping message.\n #{parsed_email.inspect}"
+      logger.error :blacklisted
```

Comment:
Do these actually get localised? What about the interpolated values?

(I'm not sure whether dumping the email into the logs is actually a good idea, but that's another question!)